src/customer/strategies/googlepay/googlepay-customer-strategy.ts

```diff
@@ -26,15 +28,17 @@ export default class GooglePayBraintreeCustomerStrategy extends CustomerStrategy
             return super.initialize(options);
         }
 
-        const { googlepaybraintree, methodId }  = options;
+        const { methodId }  = options;
+
+        const googlePayOptions = this._getGooglePayOptions(options);
```

Comment:
I believe you need to do the same thing for `GooglePaymentStrategy` as well, as it's also not done correctly.